keyboards/unicomp/classic_ultracl_post_2013/aristides_2862/readme.md

```diff
@@ -0,0 +1,27 @@
+# unicomp/classic_ultracl_post_2013/aristides_2862
+
+![unicomp/classic_ultracl_post_2013/aristides_2862](https://www.pckeyboard.com/mm5/graphics/00000001/UB4ZPHA_800x343.jpg)
```

Comment:
Please rehost these images on Imgur.

Reply:
I have rehosted some of the images on imgur, but some are linking directly to the manufacturer's website, and they're not available under some open license, so I can't just repost them.